include/libfreenect2/frame_listener.hpp

```diff
@@ -54,11 +54,17 @@ class LIBFREENECT2_API Frame
   size_t height;          ///< Number of lines in the frame.
   size_t bytes_per_pixel; ///< Number of bytes in a pixel.
   unsigned char* data;    ///< Data of the frame (aligned).
+  float exposure;         ///< Get the exposure time set by the color camera.
+  float gain;             ///< Get the gain set by the color camera.
+  float gamma;            ///< Get the gamma level set by the color camera.
 
   Frame(size_t width, size_t height, size_t bytes_per_pixel) :
     width(width),
     height(height),
-    bytes_per_pixel(bytes_per_pixel)
+    bytes_per_pixel(bytes_per_pixel),
+    exposure(30.f),
```

Comment:
To be honest, I have no idea what might be a good default value. It ranges from < 0.1 up to 66.4719085693 when it's fully covered. 

So you mean `1.f/30.f * 1000.f`?